Framework/Core/include/Framework/ASoA.h

```diff
@@ -930,6 +928,9 @@ struct RowViewCore : public IP, C... {
   }
 
  private:
+  template <class... Ts> struct overload : Ts... { using Ts::operator()...; };
```

Comment:
We already have this defined in `Framework/Foundation/include/Framework/VariantHelpers.h`

Reply:
Done.